upup/pkg/fi/cloudup/iam_builder.go

```diff
@@ -105,6 +105,38 @@ func (b *IAMPolicyBuilder) BuildAWSIAMPolicy() (*IAMPolicy, error) {
 			Action:   []string{"elasticloadbalancing:*"},
 			Resource: []string{"*"},
 		})
+
+		// Restrict the KMS permissions to only the keys that are being used
+		var set = make(map[string]bool)
```

Comment:
FYI, you can do `set := make(map[string]bool)`  There's also a StringSet implementation in the k8s code (and we already have a dependency on it).  Also I normally prefer to use more descriptive names, like `kmsKeyIDs` in this case, based on what we're putting in there.